### ISSUE TITLE
Model overfitting

### ISSUE THREAD

**Issue by Annieliaquat** [tensorflow/models#10740]
My model has training loss of 0.21 and validation loss of 0.37. Is it overfitting??

**Comment by chunduriv** [comment]
@Annieliaquat,

Yes, it is a overfitting. Thank you.
